corehq/apps/app_manager/views/forms.py

```diff
@@ -534,6 +532,11 @@ def patch_xform(request, domain, app_id, form_unique_id):
     return json_response(response_json)
 
 
+def apply_patch(patch, text):
```

Comment:
Definitely a solid improvement on readability. Not a fan of `dmp`, but I guess that's existing code that was just moved around